src/libs/actions/PersonalDetails.js

```diff
@@ -164,9 +165,10 @@ function getFromReportParticipants(reports) {
                         .sort((first, second) => first.firstName - second.firstName)
                         .map(item => item.avatar);
 
-                    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`, {icons: avatars});
+                    reportsToUpdate[`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`] = {icons: avatars};
                 }
             });
+            Onyx.mergeCollection(ONYXKEYS.COLLECTION.REPORT, reportsToUpdate);
```

Comment:
Maybe add just a bit of context so that people will understand why `mergeCollection()` is better than `merge()`